### PR TITLE
Fix mobile view

### LINKED ISSUE #527: Mobile view broken
image

Working on the fix.

### PR DESCRIPTION
**Pull request by sidharthv96**
Mobile: 
image

Desktop:
image

Fixes #527

### CODE REVIEW

**Comment by sidharthv96** [comment]
@MikeMcQuaid Should I remove the select text function or the $ prefix from install?

**Comment by MikeMcQuaid** [comment]
> @MikeMcQuaid Should I remove the select text function or the $ prefix from install?

I removed the select text already before merging the last PR. I don't think the `$` prefix needs removed as you'd already be copying only part of the line if you don't use the JS.

File: assets/css/styles.scss [hunk 1]
```diff
@@ -220,12 +220,12 @@ div .highlight {
 }
 
 .copyable {
-  display: inline-flex;
-  @include border-radius(0.4em);
-
+  display: flex;
+  justify-content: center;
+
```
Comment (MikeMcQuaid) [inline assets/css/styles.scss:225]:
```suggestion

```

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by sidharthv96** [comment]
Cool! I didn't notice that you had removed it.

**Comment by MikeMcQuaid** [comment]
Thanks again for the quick fix @sidharthv96!

**Pull request merged**